homeassistant/components/motion_blinds/cover.py

```diff
@@ -234,23 +240,69 @@ async def async_will_remove_from_hass(self):
         self._blind.Remove_callback(self.unique_id)
         await super().async_will_remove_from_hass()
 
+    async def async_scheduled_update_request(self, *_):
+        """Request a state update from the blind at a scheduled point in time."""
+        # add the last position to the list and keep the list at max 2 items
+        self._previous_positions.append(self.current_cover_position)
+        if len(self._previous_positions) > 2:
+            del self._previous_positions[: len(self._previous_positions) - 2]
+
+        await self.hass.async_add_executor_job(self._blind.Update_trigger)
+        self.async_write_ha_state()
+
+        if len(self._previous_positions) < 2 or not all(
+            self.current_cover_position == prev_position
+            for prev_position in self._previous_positions
+        ):
+            # keep updating the position @UPDATE_INTERVAL_MOVING until the position does not change.
+            async_track_point_in_time(
+                self.hass,
+                self.async_scheduled_update_request,
+                dt_util.utcnow() + timedelta(seconds=UPDATE_INTERVAL_MOVING),
+            )
+        else:
+            self._previous_positions = []
+            self._requesting_position = False
+
+    @callback
+    def async_scheduled_update_request_callback(self, now):
```

Comment:
I don't think we need this method. We can pass `async_scheduled_update_request` to `track_point_in_time`.